xcat3/plugins/control/openbmc.py

```diff
@@ -0,0 +1,83 @@
+from oslo_log import log
+from xcat3.plugins.control import base
+from xcat3.common import exception
+from xcat3.common import states
+from xcat3.common import rest
+
+import pdb
+
+LOG = log.getLogger(__name__)
+
+url_dict  = {"set_power_state" : "/xyz/openbmc_project/state/host0/attr/RequestedHostTransition",
+             "get_power_state"  : "/xyz/openbmc_project/state/host0" }
+
+data_dict = {states.POWER_COMMAND_ON   : "xyz.openbmc_project.State.Host.Transition.On",
+             states.POWER_COMMAND_OFF  : "xyz.openbmc_project.State.Host.Transition.Off",
+             states.POWER_COMMAND_RESET: "xyz.openbmc_project.State.Host.Transition.Reboot"}
+
+class OPENBMCPlugin(base.ControlInterface) :
+
+    def validate(self, node) :
+        bmc = node.control_info.get('bmc')
```

Comment:
I just use the attribute name as perl version. If change must modify node's definition. So if other parts are all ok, I will change this.